### DISCUSSION TITLE
Show case memos.web.app

### DISCUSSION THREAD

**Discussion by fabriziogiordano** [ggml-org/whisper.cpp#73] [Show and tell]
This is not an issue. I just wanted to thank you.
I create a memos web app that transcribe your memos.
Nothing fancy but with you code I can make it run on a small shared server.

Url:
https://memos.web.app/

Git:
https://github.com/fabriziogiordano/memos

![Memos - Whisper AI Web App](https://raw.githubusercontent.com/fabriziogiordano/memos/main/src/frontend/assets/images/android-chrome-96x96.png "Memos - Whisper AI Web App")

**Comment by fabriziogiordano** [comment]
Closing since this was not really an issue.
But I hope @ggerganov will check it out :)

**Reply by ggerganov** [reply]
Awesome!
I converted the issue into a discussion in case other people also find it interesting.

**Comment by jafri** [comment]
What model size is this using?

**Comment by fabriziogiordano** [comment]
Thank you @ggerganov your work is just amazing.

@jafri it is currently using the tiny model. Here is the [source](https://github.com/fabriziogiordano/memos/blob/c2c68018c45a8f0de4700fcb5c77bbb2acf9f032/src/backend/index.js#L69-L71).

This is pretty much a proof of concept. The bottle neck here is the server I use to do the transcoding. It is the cheapest server available on [Vultr](https://www.vultr.com/pricing/#cloud-compute) (ps I am not sponsored at all).

I used the tiny simply for the transcoding speed. Base was ok but a little slow for the web app. I will try to deploy a stronger server (with at least 2 CPU and more memory).


The current CPU specifications:

```
>lscpu
Architecture:                    x86_64
CPU op-mode(s):                  32-bit, 64-bit
Byte Order:                      Little Endian
Address sizes:                   40 bits physical, 48 bits virtual
CPU(s):                          1
On-line CPU(s) list:             0
Thread(s) per core:              1
Core(s) per socket:              1
Socket(s):                       1
NUMA node(s):                    1
Vendor ID:                       AuthenticAMD
CPU family:                      23
Model:                           49
Model name:                      AMD EPYC-Rome Processor
Stepping:                        0
CPU MHz:                         1996.251
BogoMIPS:                        3992.50
```
